### DISCUSSION TITLE
Qiita Advent Calendar について

### DISCUSSION THREAD

**Discussion by zacky1972** [toppers/users-forum#155] [General]
北九大の山崎進です．

Qiita Advent Calendar 2024 がオープンしました．

https://qiita.com/advent-calendar/2024

今年もTOPPERSのQiita Advent Calendar を作成するのであれば，僭越ながら寄稿したいと思います．

**Comment by hori-takeshi** [comment]
Advent Calendar 運営担当です。今年もやりますので、ぜひ寄稿よろしくお願い致します。

**Comment by hori-takeshi** [comment]
作りました！
https://qiita.com/advent-calendar/2024/toppers

後で、ANNOUNCE にも告知の記事を投稿します。
